framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -704,15 +704,13 @@ private void standardizeAtm(
      * @param context the context
      * @param localScope the local scope
      * @param useLocalScope whether {@code localScope} should be used to resolve identifiers
-     * @param atMethodSignature if true, use "#2" instead of names for formal parameters
      * @return the standardized version of the Java expression
      */
     protected String standardizeString(
             String expression,
             JavaExpressionContext context,
             TreePath localScope,
-            boolean useLocalScope,
-            boolean atMethodSignature) {
```

Comment:
Because this isn't needed, JavaExpression#atMethodSignature, isn't needed, so I deleted it.